kuma/wiki/jinja2/wiki/includes/spam_error.html

```diff
@@ -14,3 +14,10 @@
     contact the site administrators</a>.
     {% endtrans %}
 </p>
+{% if perms.wiki.add_revisionakismetsubmission %}
```

Comment:
Thanks for testing! I wasn't sure how to call this up locally to test. Can you give me some pointers on how to see this page so I can test it?